--- gwget-1.0.4/configure.ac~	2011-04-07 16:05:05.943174946 +0200
+++ gwget-1.0.4/configure.ac	2011-04-07 16:07:38.896513284 +0200
@@ -96,11 +96,11 @@
 
 EPHY_VER=
 AC_ARG_WITH([epiphany-version],
-        AC_HELP_STRING([--with-epiphany-version@<:@=1.6|1.8|1.10|2.14|2.16|2.18|2.20|2.22|2.24|2.26|2.28|2.30@:>@],
-                       [Whether to use epiphany version 1.6 or 1.8 or 1.10 or 2.14 or 2.16 or 2.18 or 2.20 or 2.22 or 2.24 or 2.26 or 2.28 or 2.30 (default 1.6)]),
+        AC_HELP_STRING([--with-epiphany-version@<:@=1.6|1.8|1.10|2.14|2.16|2.18|2.20|2.22|2.24|2.26|2.28|2.30|3.0.0@:>@],
+                       [Whether to use epiphany version 1.6 or 1.8 or 1.10 or 2.14 or 2.16 or 2.18 or 2.20 or 2.22 or 2.24 or 2.26 or 2.28 or 2.30 or 3.0 (default 1.6)]),
         [EPHY_VER="$withval"])
 
-dnl try to autodetect Epiphany 1.2, 1.4, 1.6, 1.8, 1.10, 2.14, 2.15, 2.16, 2.17, 2.18, 2.19, 2.20, 2.21, 2.22, 2.23, 2.24, 2.25, 2.26, 2.27 or 2.28 or 2.30
+dnl try to autodetect Epiphany 1.2, 1.4, 1.6, 1.8, 1.10, 2.14, 2.15, 2.16, 2.17, 2.18, 2.19, 2.20, 2.21, 2.22, 2.23, 2.24, 2.25, 2.26, 2.27 or 2.28 or 2.30 or 3.0
 EPHY_AUTODETECT="(autodetect)"
 if pkg-config --exists epiphany-2.29; then
         EPHY_VER=2.29
@@ -235,13 +235,17 @@
 	  EPIPHANY_PKGCONFIG=epiphany-2.29
           EPIPHANY_REQUIRED=2.30
 	  EPIPHANY_SHORTVER=230
+elif test "x$EPHY_VER" = "x3.0"; then
+	  EPIPHANY_PKGCONFIG=epiphany-3.0
+	  EPIPHANY_REQUIRED=3.0
+	  EPIPHANY_SHORTVER=300
 else
      	AC_MSG_RESULT([no epiphany installed])
 fi
 
 if test -n "$EPIPHANY_SHORTVER" ; then
 AC_DEFINE_UNQUOTED(EPHY_VERSION, $EPIPHANY_SHORTVER,
-       [Define to the Epiphany version (12, 14, 16, 18, 110, 214, 215, 216, 217, 218, 219, 220, 221, 222, 223, 224, 225, 226, 227 or 228 or 230)])
+       [Define to the Epiphany version (12, 14, 16, 18, 110, 214, 215, 216, 217, 218, 219, 220, 221, 222, 223, 224, 225, 226, 227 or 228 or 230 or 3.0)])
 AC_MSG_RESULT([$EPHY_VER $EPHY_AUTODETECT])
 
 LIBGLIB_REQUIRED=2.4.0
