diff -urN aaa/gwget-0.98.1/configure.in gwget-0.98.1/configure.in
--- aaa/gwget-0.98.1/configure.in	2006-05-07 20:06:52.000000000 +0200
+++ gwget-0.98.1/configure.in	2006-08-13 12:38:05.614505000 +0200
@@ -99,9 +99,11 @@
                        [Whether to use epiphany version 1.6 or 1.8 or 1.10 or 2.14 (default 1.6)]),
         [EPHY_VER="$withval"])
 
-dnl try to autodetect Epiphany 1.2, 1.4, 1.6, 1.8, 1.10 or 2.14
+dnl try to autodetect Epiphany 1.2, 1.4, 1.6, 1.8, 1.10, 2.14 or 2.15
 EPHY_AUTODETECT="(autodetect)"
-if pkg-config --exists epiphany-2.14; then
+if pkg-config --exists epiphany-2.15; then
+        EPHY_VER=2.15
+elif pkg-config --exists epiphany-2.14; then
         EPHY_VER=2.14
 elif pkg-config --exists epiphany-1.10; then
         EPHY_VER=1.10
@@ -144,13 +146,17 @@
           EPIPHANY_PKGCONFIG=epiphany-2.14
           EPIPHANY_REQUIRED=2.14
           EPIPHANY_SHORTVER=214
+elif test "x$EPHY_VER" = "x2.15"; then
+          EPIPHANY_PKGCONFIG=epiphany-2.15
+          EPIPHANY_REQUIRED=2.15
+          EPIPHANY_SHORTVER=215
 else
      	AC_MSG_RESULT([no epiphany installed])
 fi
 
 if test -n "$EPIPHANY_SHORTVER" ; then
 AC_DEFINE_UNQUOTED(EPHY_VERSION, "$EPIPHANY_SHORTVER",
-       [Define to the Epiphany version (12, 14, 16, 18, 110 or 214)])
+       [Define to the Epiphany version (12, 14, 16, 18, 110, 214 or 215)])
 AC_MSG_RESULT([$EPHY_VER $EPHY_AUTODETECT])
 
 LIBGLIB_REQUIRED=2.4.0
