diff -urN gwget-0.99/configure.in gwget-0.99.new/configure.in
--- gwget-0.99/configure.in	2008-03-18 16:17:20.000000000 +0100
+++ gwget-0.99.new/configure.in	2008-03-18 16:44:27.000000000 +0100
@@ -194,6 +194,7 @@
                   glib-2.0 >= $LIBGLIB_REQUIRED \
                   gmodule-2.0 \
                   gtk+-2.0 >= $LIBGTK_REQUIRED \
+                  libgnomeui-2.0 >= $LIBGNOMEUI_REQUIRED \
                   $EPIPHANY_PKGCONFIG >= $EPIPHANY_REQUIRED \
 		  )
 AC_SUBST(EPIPHANY_DEPENDENCY_CFLAGS)
