--- gwget-0.99/configure.in.orig	2007-10-12 00:49:29.207963603 +0000
+++ gwget-0.99/configure.in	2007-10-12 00:51:27.189964708 +0000
@@ -103,9 +103,11 @@
                        [Whether to use epiphany version 1.6 or 1.8 or 1.10 or 2.14 (default 1.6)]),
         [EPHY_VER="$withval"])
 
-dnl try to autodetect Epiphany 1.2, 1.4, 1.6, 1.8, 1.10, 2.14, 2.15, 2.16, 2.17 or 2.18
+dnl try to autodetect Epiphany 1.2, 1.4, 1.6, 1.8, 1.10, 2.14, 2.15, 2.16, 2.17, 2.18 or 2.20
 EPHY_AUTODETECT="(autodetect)"
-if pkg-config --exists epiphany-2.18; then
+if pkg-config --exists epiphany-2.20; then
+	EPHY_VER=2.20
+elif pkg-config --exists epiphany-2.18; then
 	EPHY_VER=2.18
 elif pkg-config --exists epiphany-2.17; then
         EPHY_VER=2.17
@@ -170,6 +172,10 @@
 	  EPIPHANY_PKGCONFIG=epiphany-2.18
           EPIPHANY_REQUIRED=2.18
 	  EPIPHANY_SHORTVER=218
+elif test "x$EPHY_VER" = "x2.20"; then
+	  EPIPHANY_PKGCONFIG=epiphany-2.20
+          EPIPHANY_REQUIRED=2.20
+	  EPIPHANY_SHORTVER=220
 else
      	AC_MSG_RESULT([no epiphany installed])
 fi
