diff -urN gwget-0.14.orig/gwget.desktop.in gwget-0.14/gwget.desktop.in
--- gwget-0.14.orig/gwget.desktop.in	2004-09-02 16:20:16.599336432 +0200
+++ gwget-0.14/gwget.desktop.in	2004-09-02 16:24:53.948173000 +0200
@@ -1,15 +1,18 @@
 Encoding=UTF-8
 Name=Gwget Download Manager
 GenericName=Download Manager
+GenericName[pl]=Menadżer pobierania
 Comment=Download files from the Internet
+Comment[pl]=Pobieraj pliki z Internetu
 Exec=gwget
 StartupNotify=true
 Icon=gwget.png
 Terminal=false
 X-MultipleArgs=false
 Type=Application
-Categories=Application;Network;
+Categories=GTK;GNOME;Network;FileTransfer;
 X-GNOME-Bugzilla-Bugzilla=GNOME
 X-GNOME-Bugzilla-Product=gwget
 X-GNOME-Bugzilla-Component=General
 X-GNOME-Bugzilla-Version=@VERSION@
+# vi: encoding=utf-8
