diff -urN gwget-0.98.2/configure.in gwget-0.98.2.new/configure.in
--- gwget-0.98.2/configure.in	2006-11-08 19:49:44.000000000 +0100
+++ gwget-0.98.2.new/configure.in	2007-03-15 18:27:07.000000000 +0100
@@ -101,9 +101,13 @@
                        [Whether to use epiphany version 1.6 or 1.8 or 1.10 or 2.14 (default 1.6)]),
         [EPHY_VER="$withval"])
 
-dnl try to autodetect Epiphany 1.2, 1.4, 1.6, 1.8, 1.10, 2.14, 2.15 or 2.16
+dnl try to autodetect Epiphany 1.2, 1.4, 1.6, 1.8, 1.10, 2.14, 2.15, 2.16, 2.17 or 2.18
 EPHY_AUTODETECT="(autodetect)"
-if pkg-config --exists epiphany-2.16; then
+if pkg-config --exists epiphany-2.18; then
+        EPHY_VER=2.18
+elif pkg-config --exists epiphany-2.17; then
+        EPHY_VER=2.17
+elif pkg-config --exists epiphany-2.16; then
         EPHY_VER=2.16
 elif pkg-config --exists epiphany-2.15; then
         EPHY_VER=2.15
@@ -158,13 +162,21 @@
           EPIPHANY_PKGCONFIG=epiphany-2.16
           EPIPHANY_REQUIRED=2.16
           EPIPHANY_SHORTVER=216
+elif test "x$EPHY_VER" = "x2.17"; then
+          EPIPHANY_PKGCONFIG=epiphany-2.17
+          EPIPHANY_REQUIRED=2.17
+          EPIPHANY_SHORTVER=217
+elif test "x$EPHY_VER" = "x2.18"; then
+          EPIPHANY_PKGCONFIG=epiphany-2.18
+          EPIPHANY_REQUIRED=2.18
+          EPIPHANY_SHORTVER=218
 else
      	AC_MSG_RESULT([no epiphany installed])
 fi
 
 if test -n "$EPIPHANY_SHORTVER" ; then
 AC_DEFINE_UNQUOTED(EPHY_VERSION, "$EPIPHANY_SHORTVER",
-       [Define to the Epiphany version (12, 14, 16, 18, 110 ,214, or 215)])
+       [Define to the Epiphany version (12, 14, 16, 18, 110 ,214, 215, 216, 217, or 218)])
 AC_MSG_RESULT([$EPHY_VER $EPHY_AUTODETECT])
 
 LIBGLIB_REQUIRED=2.4.0
