--- configure.in.org	2008-03-18 10:25:14.132459386 +0100
+++ configure.in	2008-03-18 10:27:53.866762265 +0100
@@ -103,7 +103,9 @@
 
 dnl try to autodetect Epiphany 1.2, 1.4, 1.6, 1.8, 1.10, 2.14, 2.15, 2.16, 2.17 or 2.18
 EPHY_AUTODETECT="(autodetect)"
-if pkg-config --exists epiphany-2.18; then
+if pkg-config --exists epiphany-$EPHY_VER; then
+	:
+elif pkg-config --exists epiphany-2.18; then
 	EPHY_VER=2.18
 elif pkg-config --exists epiphany-2.17; then
         EPHY_VER=2.17
@@ -171,7 +173,9 @@
           EPIPHANY_REQUIRED=2.18
 	  EPIPHANY_SHORTVER=218
 else
-     	AC_MSG_RESULT([no epiphany installed])
+	  EPIPHANY_PKGCONFIG=epiphany-${EPHY_VER}
+	  EPIPHANY_REQUIRED=${EPHY_VER}
+	  EPIPHANY_SHORTVER=`echo $EPHY_VER | tr -d .`
 fi
 
 if test -n "$EPIPHANY_SHORTVER" ; then
