--- data/gwget.desktop.in.in.orig	2009-02-21 17:20:59.250100730 +0100
+++ data/gwget.desktop.in.in	2009-02-21 17:23:29.890083149 +0100
@@ -2,8 +2,11 @@
 Version=1.0
 Encoding=UTF-8
 _Name=Gwget Download Manager
+_Name[pl]=Menadżer pobierania Gwget
 _GenericName=Download Manager
+_GenericName=Menadżer pobierania
 _Comment=Download files from the Internet
+_Comment=Pobiera pliki z internetu
 Exec=gwget
 StartupNotify=true
 Icon=gwget.png
